app/views/job_invocations/_tab_hosts.html.erb

```diff
@@ -38,7 +38,7 @@
     </tbody>
   </table>
 
-  <%= will_paginate_with_info @hosts %>
+  <%= will_paginate_with_info @hosts, :'data-no-turbolink' => 'true', :container => true %>
```

Comment:
This is what allows the React component to mount when you click on next/prev page on the hosts table